ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -95,24 +95,47 @@ public struct AvatarGroup: View, TokenizedControlView {
     public typealias TokenSetKeyType = AvatarGroupTokenSet.Tokens
     @ObservedObject public var tokenSet: AvatarGroupTokenSet
 
-    /// Creates and initializes a SwiftUI AvatarGroup.
+    /// Creates and initializes a SwiftUI `AvatarGroup`.
     /// - Parameters:
     ///   - style: The style of the avatar group.
     ///   - size: The size of the avatars displayed in the avatar group.
+    ///   - avatarCount: The number of Avatars in this group.
+    ///   - maxDisplayedAvatars: Caps the number of displayed avatars and shows the remaining not displayed in the overflow avatar.
+    ///   - overflowCount: Adds to the overflow count in case the calling code did not provide all the avatars, but still wants to convey more
+    ///                    items than just the remainder of the avatars that could not be displayed due to the maxDisplayedAvatars property.
+    ///   - isUnread: Show a top-trailing aligned unread dot when set to true.
+    ///   - avatarBuilder: A ViewBuilder that creates an `Avatar` for a given index.
     public init(style: MSFAvatarGroupStyle,
-                size: MSFAvatarSize) {
+                size: MSFAvatarSize,
+                avatarCount: Int = 0,
+                maxDisplayedAvatars: Int = 5,
```

Comment:
I assume this is non-zero so we always show something if a implementation doesn't specify it, and 5 is just an arbitrary value picked?

Reply:
Good catch--that was originally just for testing. The default should be `Int.max`.